data/test/_main.cfg

```diff
@@ -74,6 +74,7 @@
 {test/scenarios/wml_tests/WesnothFormulaLanguage}
 
 # Load test unit wml
+{test/scenarios/traits_exclude_include.cfg}
```

Comment:
I think the correct fix is instead to move this file into a different folder. @Pentarctagon should be able to help you figure out where it goes.

Reply:
I was thinking the same thing, where should I put it @Pentarctagon ?

Reply:
[trait] is under UnitsWML on the wiki, so it'd be https://github.com/wesnoth/wesnoth/tree/master/data/test/scenarios/wml_tests/UnitsWML